keyvalue/client/src/main/java/alluxio/client/keyvalue/BasePayloadWriter.java

```diff
@@ -40,7 +40,8 @@
    * @param out the stream to output payload
    */
   BasePayloadWriter(AbstractOutStream out) {
-    mOutStream = Preconditions.checkNotNull(out);
+    mOutStream = Preconditions.checkNotNull(out,
+            "Cannot create a BasePayloadWriter from a null AbstractOutStream");
```

Comment:
Hi @pilife , I think the following is good:
`mOutStream = Preconditions.checkNotNull(out, "out")`

Reply:
why~ I just simulate the [URIStatus class]'s Construction~